server/server.go

```diff
@@ -327,6 +332,9 @@ func (s *Server) setupServerEntryPoint(newServerEntryPointName string, newServer
 		serverMiddlewares = append(serverMiddlewares, ipWhitelistMiddleware)
 		serverInternalMiddlewares = append(serverInternalMiddlewares, ipWhitelistMiddleware)
 	}
+	if s.osioMiddleware != nil {
+		serverMiddlewares = append(serverMiddlewares, s.osioMiddleware)
+	}
```

Comment:
OSIO middleware added into serverMiddlewares.
Traefik has two types of middleware - serverMiddlewares and serverInternalMiddlewares.
Our OSIO middleware seems serverMiddlewares.